src/components/Logo.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from "react";
+
+export const Logo = (props, { }) => {
```

Comment:
I think you could have gone with `just = ( )` since you're not using the 2 params